cv_connector/CV_type cv_type
---
sensor_msgs/CameraInfo camera_info
sensor_msgs/PointCloud2 pointcloud
sensor_msgs/Image rgb_image
sensor_msgs/Image depth_image

string result
bool sucess